python_on_whales/components/container/cli_wrapper.py

```diff
@@ -626,6 +626,7 @@ def create(
         systemd: Optional[Union[bool, Literal["always"]]] = None,
         tmpfs: List[ValidPath] = [],
         tty: bool = False,
+        tz: Optional[str] = None,
```

Comment:
Missing update to docstring. The PR description says "as supported by podman and docker", but is it supported by docker? I can't see it at https://docs.docker.com/reference/cli/docker/container/run/#options

Reply:
In fact you've implemented the tests to only run with podman, was that intentional? I think just need to say in the docstring that it's podman only, and update the PR description for clarity.

Reply:
I was mistaken in my PR description, so I'll make this clear.